src/main/java/org/fenixedu/academic/ui/struts/action/candidacy/degreeChange/DegreeChangeCandidacyProcessDA.java

```diff
@@ -511,7 +511,10 @@ protected Spreadsheet buildIndividualCandidacyReport(final Spreadsheet spreadshe
         row.setCell(degreeChangeIndividualCandidacyProcess.getPersonalDetails().getName());
         row.setCell(degreeChangeIndividualCandidacyProcess.getPersonalDetails().getIdDocumentType().getLocalizedName());
         row.setCell(degreeChangeIndividualCandidacyProcess.getPersonalDetails().getDocumentIdNumber());
-        row.setCell(degreeChangeIndividualCandidacyProcess.getPersonalDetails().getCountry().getCountryNationality().getContent());
+        if (degreeChangeIndividualCandidacyProcess.getPersonalDetails().getCountry().getCountryNationality().getContent() != null)
+            row.setCell(degreeChangeIndividualCandidacyProcess.getPersonalDetails().getCountry().getCountryNationality().getContent());
+        else
+            row.setCell("-");
```

Comment:
missing { and } in if - else statement.
The NPE occurs when dereferencing ``getCountry`` I think.